docs/unlocked_package.rst

```diff
@@ -3,105 +3,139 @@ Release an Unlocked Package
 
 While CumulusCI was originally created to develop managed packages, it can also be used to develop and release `unlocked packages <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_intro.htm>`_.
 
-.. warning::
-
-    We do not currently support setting the Ancestor Id for a 2GP package.
-
 
 Prerequisites
 -------------
+This section assumes:
 
-To create unlocked package versions, complete these steps.
+* :doc:`CumulusCI is installed <get_started>` on your computer.
+* A Salesforce project has been configured for use with CumulusCI.
```

Comment:
link to https://cumulusci.readthedocs.io/en/feature-2gp-docs/get_started.html#project-initialization ?